Source/AH/Initialize.lua

```diff
@@ -0,0 +1,20 @@
+function Auctionator.AH.Initialize()
+  if Auctionator.AH.Internals ~= nil then
+    return
+  end
+  Auctionator.AH.Internals = {}
+
+  if Auctionator.Config.Get(Auctionator.Config.Options.SILENCE_AUCTION_ERRORS) then
+    -- Silence excessive errors
+    ERR_AUCTION_DATABASE_ERROR = ""
```

Comment:
Do we ever want to turn this back on? (I think this is fine, just trying to think through ramifications...)

Reply:
That's up to you, probably not.

Reply:
Its an invisible option so we can leave it there indefinitely, with no penalty